#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_correct-Stumpwm-StumpWM-in-README.patch.dpatch by  <luca@pca.it>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct s/Stumpwm/StumpWM/ in README

@DPATCH@

diff -Naur stumpwm/README stumpwm-debian-02/README
--- stumpwm/README	2006-05-18 00:14:59.000000000 -0500
+++ stumpwm-debian-02/README	2006-05-19 12:26:57.000000000 -0500
@@ -1,7 +1,7 @@
 The Stump Window Manager
 ------------------------
 
-Stumpwm is a window manager written entirely in Common Lisp. It
+StumpWM is a window manager written entirely in Common Lisp. It
 attempts to be highly customizable while relying entirely on the
 keyboard for input. You will not find buttons, icons, title bars, tool
 bars, or any of the other conventional GUI widgets. 
